content/help/faq/distributed-tracing/minimal-requirements.md

```diff
@@ -0,0 +1,6 @@
+---
+title: What is the minimal feasible Istio configuration required for Distributed Tracing?
+weight: 12
+---
+
+The Istio minimal profile with tracing enabled is all that is required for Istio to integrate with Zipkin API-compatible tracing backends.
```

Comment:
don't you also need to enable the telemetry component?  or is this directly from sidecar to Zipkin API-compatible tracing backends without mixer?

Reply:
Mixer is not required for tracing, especially for minimal deploys.